compiler/testData/codegen/box/defaultArguments/kt52702.kt

```diff
@@ -0,0 +1,23 @@
+// Check that local variables for inline functions and inline default lambdas start
```

Comment:
Please ignore the test for `JS` backend, otherwise `JsCodegenBoxTestGenerated.DefaultArguments.testKt52702` fails